doc/Developing.md

```diff
@@ -50,7 +50,16 @@ tab with the frontend.
 
 ## Run the electron-based example application
 
-It can also be started with:
+From the root directory run:
+
+    npm run rebuild:electron
+
+This command rebuilds native node packages against the version of node used by electron.
+To rollback changes run:
+
+    npm run rebuild:web
+
+It can also be started with from examples/electron directory:
```

Comment:
OK yes I was curious about that so this means that since with hoisting we have only 1 node-modules we can only have 1 example at the time without rebuilding :(
But yes sounds better to me in the web section thx